components/engine_traits/src/lib.rs

```diff
@@ -119,16 +119,6 @@
 //!   trait IOLimiter { }
 //!   ```
 //!
-//!   ```
```

Comment:
I think it is better to use `ignore` after the triple backtick, rather than deleting the example

Reply:
I heart that this impl should have been removed in master.

Reply:
Thanks for your advice.

Reply:
FYI, there is no IOLimiter trait in engine_traits any more.

Reply:
OK, but I think we can still retain it as a example here.

Reply:
@mahjonp While `IOLimiter` has been removed there should still be other existing `Xxxx`/`XxxxExt` trait pairs.